src/ethlance/ui/graphql.cljs

```diff
@@ -7,7 +7,6 @@
     [district.shared.async-helpers :refer [promise->]]
     [ethlance.ui.util.component :refer [>evt]]
     [re-frame.core :as re-frame]
-    [re-frame.core :as re]
```

Comment:
Let's settle that everywhere we'll be using re-frame as `[re-frame.core :as re]`